### ISSUE TITLE
Investigate client post message window parent undefined check 

### ISSUE THREAD

**Issue by ndricimrr** [SAP/luigi#2794]
```
    if (this.origin) {
      window.parent.postMessage(msg, this.origin);
    } else {
      console.warn(
        'There is no target origin set. You can specify the target origin by calling LuigiClient.setTargetOrigin("targetorigin") in your micro frontend.'
      );
    }
```
The above assumes that the windows **parent** always exists, but this is not always the case. When window parent is gone this produces the following error: 
![Screenshot 2022-06-30 at 18.13.04.png](https://images.zenhubusercontent.com/5ddce54970febb0001f9c812/ea0e2c08-67d5-43fc-9b61-ffec0d95e8a8)
Might need to add a check to never attempt to post a message if the windows parent doesnt exist, to avoid this ugly error from appearing. Would make debugging easier too.

**Comment by hardl** [comment]
Agree, not nice to have it uncaught, replace with a log.debug()

**Comment by ndricimrr** [comment]
Already resolved in #2888
